docs/VOLA_API.md

```diff
@@ -0,0 +1,59 @@
+Api to communicate with Vola
+----------------------------
+----------------------------
+The purpose of building this API was to provide information about the events
+in VMS to Vola so that their volunteers could participate in them. Presently,
+these fields of events are sent to Vola via the API::
+
+1. Event Name - Title of the event going to be held.
+2. Start Date - The date from which the event would start.
+3. End Date - The date on which the event would end.
+4. Description - The details of the event
+5. Address - The address on which event would take place.
+6. City - The city where event would take place.
+7. State - The state where event would take place.
+8. Country - The country where event would take place.
+9. Venue - The location of the event.
+
+ Users can send in a ``GET`` or a ``POST`` request to `https://localhost/event/api/v1/request_event_data/ <https://localhost/event/api/v1/request_event_data/>`_ to access the event data.
```

Comment:
`_ to access the event data.` Remove underscore.
Did you want a link here? Why don't you just use angular brackets? It's not code.